code/modules/jobs/job_types/_job.dm

```diff
@@ -446,6 +447,7 @@
 		player_client.prefs.safe_transfer_prefs_to(src, TRUE, is_antag)
 		if (require_human && !ishumanbasic(src))
 			set_species(/datum/species/human)
+			dna.species.roundstart_changed = TRUE
```

Comment:
why not do this as part of set_species?

Reply:
Because other things use set species that aren't roundstart